app/screens/send-bitcoin-screen/send-bitcoin-destination-screen.tsx

```diff
@@ -240,14 +271,37 @@ const SendBitcoinDestinationScreen: React.FC<Props> = ({ route }) => {
     }
   }, [destinationState, goToNextScreenWhenValid, navigation, setGoToNextScreenWhenValid])
 
-  const initiateGoToNextScreen = useMemo(() => {
-    return async () => {
-      if (validateDestination) {
-        await validateDestination(destinationState.unparsedDestination)
-        setGoToNextScreenWhenValid(true)
+  // setTimeout here allows for the main JS thread to update the UI before the long validateDestination call
```

Comment:
how would the UI been updated? Is it the opacity of the button that would react for instance? or is it also that the text should be shown while it's been validated?